docs/source/examples/supplychain/supplychain_transaction_family.rst

```diff
@@ -43,61 +43,121 @@ Agent
 Agents represent owners or custodian of goods in the system. All agents must
 be registered in the system before they can interact with records.
 
-.. literalinclude:: ../../../../families/supplychain/protos/agent.proto
```

Comment:
Hmm. I prefer the include model. I understand the issue with the offsets. Seems to me that the least evil option is to do the include without the lines stripped.